### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'DEV')

### ISSUE THREAD

**Issue by martialanouman** [nksaraf/vinxi#244]
Hi there!

I have have this error when i use `lazyRoute` function from `@vinxi/react`.
It seems that `import.meta.env` is undefined for some reason.

```shell
vinxi starting dev server
(!) Could not auto-determine entry point from rollupOptions or html files and there are no explicit optimizeDeps.include patterns. Skipping dependency pre-bundling.

  ➜ Local:    http://localhost:3000/
  ➜ Network:  use --host to expose

TypeError: Cannot read properties of undefined (reading 'DEV')
    at file:///Users/dewz/Documents/Sources/React/vinxi-meta/node_modules/.pnpm/@vinxi+react@0.2.2/node_modules/@vinxi/react/lazy-route.js:22:23
    at lazyInitializer (/Users/dewz/Documents/Sources/React/vinxi-meta/node_modules/.pnpm/react@18.2.0/node_modules/react/cjs/react.development.js:1357:20)
    at renderLazyComponent (/Users/dewz/Documents/Sources/React/vinxi-meta/node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom-server.node.development.js:6000:19)
    at renderElement (/Users/dewz/Documents/Sources/React/vinxi-meta/node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom-server.node.development.js:6095:11)
    at renderNodeDestructiveImpl (/Users/dewz/Documents/Sources/React/vinxi-meta/node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom-server.node.development.js:6170:11)
    at renderNodeDestructive (/Users/dewz/Documents/Sources/React/vinxi-meta/node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom-server.node.development.js:6142:14)
    at renderNode (/Users/dewz/Documents/Sources/React/vinxi-meta/node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom-server.node.development.js:6325:12)
    at renderSuspenseBoundary (/Users/dewz/Documents/Sources/React/vinxi-meta/node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom-server.node.development.js:5657:5)
    at renderElement (/Users/dewz/Documents/Sources/React/vinxi-meta/node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom-server.node.development.js:6060:11)
    at renderNodeDestructiveImpl (/Users/dewz/Documents/Sources/React/vinxi-meta/node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom-server.node.development.js:6170:11)
```

Code snippet
```typescript
const routes = fileRoutes.map(route => ({
	...route,
	Component: lazyRoute(
		route['$component'],
		clientManifest,
		serverManifest
	),
}))
```

Packages:
```json
"@vinxi/react": "^0.2.2",
"@vinxi/server-functions": "^0.3.2",
"react": "^18.2.0",
"react-dom": "^18.2.0",
"vinxi": "^0.3.10"
```

**Comment by nksaraf** [comment]
i have seen this issue at places, i think it has to do with vite not transpiling that package, i think this needs to be fixed at our end. Ill take a look

**Comment by martialanouman** [comment]
Hi @nksaraf ,

any update on this ?

**Comment by nksaraf** [comment]
fixed by #265
